config/locales/en.yml

```
@@ -1,15 +1,34 @@
 en:
-  category: "Category"
   comment: "Comment"
   description: "Description"
+  time:
+    formats:
+      default: "%H:%M %d %B, %Y "
```

Chỗ này có phải dư không á huynh
```suggestion
      default: "%H:%M %d %B, %Y"
```